.travis.yml

```
@@ -10,6 +10,13 @@ matrix:
   include:
     - python: 3.6
       env: BUILD_DOCS=1
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Looks like all builds require `sudo` (see the `sudo apt-get update` below).

... and poking around a bit further, there are no other `sudo` calls. Is there a not-so-obvious script requiring sudo here?

---

No idea! Copied from `happi`, which has a link to this: https://github.com/travis-ci/travis-ci/issues/9815, which I actually did not read.

Maybe @teddyrendahl knows

---

Looks like with `sudo: false` we will have some strange silent failure according to the comment/issue chain:

https://github.com/travis-ci/travis-ci/issues/10110

I do no understand however that we have `sudo: false` set at the top of the Travis and then the first command we call requires `sudo`...